server/sonar-server/src/main/java/org/sonar/server/projectbranch/ws/ShowAction.java

```diff
@@ -127,11 +126,10 @@ private BranchDto getBranch(DbSession dbSession, String uuid) {
     return branch.get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "branch.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655)